upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,19 @@
 == Migration Changes
 
+=== Migrating to 8.0.2
+
+==== More Authentication flows changes
+
+REQUIRED and ALTERNATIVE executions not supported at same flow::
+    In previous version, it was possible to have REQUIRED and ALTERNATIVE executions in the same authentication flow at the same level.
+    There were some issues with this approach and we did the refactoring in the Authentication SPI, which means that this is not considered
+    valid anymore. If ALTERNATIVE and REQUIRED executions are configured at the same level, the ALTERNATIVE executions are considered disabled.
+    So when migrating to the newest version, your existing authentication flows will be automatically migrated preserved the same behavior as existed in the previous version.
+    If they contain ALTERNATIVE executions after some REQUIRED executions, then the ALTERNATIVE executions will be added to the separate REQUIRED subflow. This
```

Comment:
One little comment from my side: I am struggling a bit with the phrase "ALTERNATIVE executions *after* some REQUIRED executions". The "after" is misleading as the position (but not the level) doesn't really matter (see my code PR comment).

Reply:
@thielef Thanks! I've changed the sentence to this:
```
-    If they contain ALTERNATIVE executions after some REQUIRED executions, then the ALTERNATIVE executions will be added to the separate REQUIRED subflow. This
+    If they contain ALTERNATIVE executions at the same level as some REQUIRED executions, then the ALTERNATIVE executions will be added to the separate REQUIRED subflow. This
```

Reply:
@mposolda Looks 👍 now